packages/waku/src/lib/types.ts

```diff
@@ -100,3 +101,5 @@ export type HandlerRes = {
   headers?: Record<string, string | string[]>;
   status?: number;
 };
+
+export type Method = (typeof METHODS)[number];
```

Comment:
Can you move this and `METHODS` into create-pages.ts for now?
In the future, this might be changed, but for now, lower level api below create-pages just use `string` for method.